test/index.js

```diff
@@ -296,16 +296,15 @@ describe('.dest()', function() {
 
   it('outputs files where the base ends in a separator', function(done) {
```

Comment:
The test was written because we ensured base ended in a separator when passed through (there was a bug in some version of vinyl or vinyl-fs, I think).  Either this test should be removed or named something like "outputs files where the base does not end in a separator, even when exists initially".  That logic is now completely in vinyl, so I think removing it makes more sense.